src/components/homePage/Speakers.vue

```diff
@@ -1,13 +1,26 @@
 <template>
 <div id="speakers">
 	<h2>CUTC 2020</h2>
-	<!-- TAB LAYOUT -->
+	<div id="TABS">
```

Comment:
Could you keep IDs/classes `all-lowercase-with-dashes` like so if possible? Helps keep everything consistent.

Reply:
yup, i'll change that and make a new pr in a bit

Reply:
No need to make a new PR, if you push changes onto this branch they'll become part of this PR. dw :)

Reply:
oh yes, that's what i meant. it should be changed along with the carousel and pushed now